### PR TITLE
Fixing worldserver.exe errors

### PR DESCRIPTION
**Pull request by DantestyleXD**
**Changes proposed:**
Resolve some error from worldserver.exe:

- Creature Loot Entry = 1
- Wrong Skinning entry for Loot Template
- Wrong values for a condition

**Target branch(es):**
1.x

**Issues addressed:**
#96

**Tests performed:**
Tested ingame and improved for no more errors

**Known issues and TODO list:**
- TODO: 
    Fix issue about quest 10409 (Need to recreate all event),
    Fix issue about script 'spell_midnight_fixate' (Maybe move script from boss script to spell script)

----------------------------
Server: Local (Repo AzerothCore)
Last commit hash tested: 3761e9d8433f4d031b3477b787134a1d868a8971
O.S.: Windows 10
Compiler: MS Visual Studio 2013

### CODE REVIEW

File: data/sql/updates/world/2016_08_19_00.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/*
+Reference some issues: one problem #96
+Fixing worldserver.exe errors
+
+Server: Local (Repo AzerothCore)
+Last commit hash tested: 3761e9d8433f4d031b3477b787134a1d868a8971
+O.S.: Windows 10
+Compiler: MS Visual Studio 2013
+*/
```
Comment (ShinDarth) [inline data/sql/updates/world/2016_08_19_00.sql]: those comments should be placed into the PR description, not into the code itself

**Comment by ShinDarth** [comment]
well done

**Comment by ShinDarth** [comment]
@DantestyleXD can you clean the commits?

try:
https://github.com/azerothcore/azerothcore-wotlk/wiki/Syncing-your-fork

then:
`git rebase upstream master`

**Comment by DantestyleXD** [comment]
I did it like your instructions but not change here on Github (local works).

**Comment by ShinDarth** [comment]
there is something wrong with the commits: https://github.com/azerothcore/azerothcore-wotlk/pull/101/commits

**Comment by DantestyleXD** [comment]
I will recreate the branch and a pull request

**Pull request closed without merging**